tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java

```diff
@@ -170,11 +170,14 @@ public void visit(HttpListener listener) {
         String allowedMethods = Constant.BLANG_METHOD_GET;
         if (listener.getAllowedMethods() != null) { //TODO: allowedmethods can come comma separated. Handle that here.
             allowedMethods = listener.getAllowedMethods();
+            String[] bits = allowedMethods.split(",");
+            for (String method : bits) {
+                /*Create an annotation without attribute values*/
```

Comment:
Can't we use a single line comment here ?

Reply:
That comment is no more there. https://github.com/arukshani/product-ei/commit/cfdd25ac11ed12e0a02e2579579e65b1cac4e088#diff-a7c35609bfca9af1252baf6d499e26feR153